src/drawing_utilities.ts

```diff
@@ -0,0 +1,148 @@
+/* 
+This file is designed to hold all the functions 
+required for drawing to the canvas
+*/
+
+import { ULabelAnnotation } from './annotation';
+
+/*recieves a base color and applies a gradient to it based on its confidence
+The reason for passing in the get confidence function is so that we can apply
+a gradient based on diffrent confidence statistics if we choose to do so*/
+export function apply_gradient(
+    annotation_object: ULabelAnnotation, 
+    base_color: string, 
+    get_annotation_confidence: Function, 
+    gradient_maximum: number) {
+        
+    //if the gradient toggle is checked off, then don't apply a gradient
```

Comment:
it's probably good practice to have a space after the `//` starting a comment and to capitalize, but I it doesn't have to be done on everything for this PR, seeing as a linter can do it automatically down the road

Reply:
Okay, I'll keep it in mind, but I won't worry about it for this pr